integration-cli/docker_cli_build_test.go

```diff
@@ -1568,7 +1569,18 @@ func TestBuildAddBadLinks(t *testing.T) {
 	}
 	defer os.RemoveAll(tempDir)
 
-	symlinkTarget := fmt.Sprintf("/../../../../../../../../../../../..%s", tempDir)
+	var symlinkTarget string
```

Comment:
@LK4D4 you're right. it might help a bit but we still need to get the volume name `c:` somehow.